front/src/components/Chat/Chat.tsx

```diff
@@ -0,0 +1,83 @@
+import IChat from '../../types/chat/IChat';
+import IChatDetails from '../../types/chat/IChatDetails';
+
+import { useEffect, useRef, useState } from 'react';
+import { getChatById } from '../../api/chat';
+import { ScrollArea, Text } from '@mantine/core';
+import IMessage from '../../types/chat/IMessage';
+
+interface IChatProps {
+  chat: IChat | null;
```

Comment:
Is this equivalent to "chat? : IChat" ?

Reply:
Maybe, I mean, I guess ?...

Reply:
Turns out it does not. The latter is equivalent to `IChat | undefined`.